src/commands/jetpack-modules.php

```diff
@@ -24,7 +24,7 @@ protected function execute( InputInterface $input, OutputInterface $output ) {
 
 		$api_helper = new API_Helper;
 
-		$site = $api_helper->call_wpcom_api( 'rest/v1.1/sites/' . $site_domain, array() );
+		$site = $api_helper->call_wpcom_api( 'rest/v1.1/sites/' . $site_domain . '?fields=ID', array() );
```

Comment:
Good to know the performance trick also works on other endpoints

Reply:
Yes! It won't help much in this case, but best practice to limit memory size I think. I just wish it worked on the `jetpack-blogs` endpoint 😞